### PR TITLE
MRCD8-175 visitor node

### PR DESCRIPTION
**Pull request by josephgknox**
# READY FOR REVIEW

# Summary
- Visitor node styles
- Updated Gruntfile.js for stanford_basic alongside MRC theme and recompiled CSS to fix theme issue

# Needed By (Date)
- End of sprint (2/1)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Test the node display
3. Verify that Decanter grid classes are now working

# Affected Projects or Products
- MRC
- stanford_mrc

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-175

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: themes/math_research_center/patterns/templates/node-simple/scss/node-simple.scss [hunk 1]
```diff
@@ -1,2 +1,12 @@
...
+///
+/// SIMPLE NODE STYLES
+///
+
+// Roll up file for all node-specific
+// styles using the node-simple template.
+
+// Visitor Node
+@import
+  'scss/node/stanford_visitor';
```
Comment (pookmish) [inline themes/math_research_center/patterns/templates/node-simple/scss/node-simple.scss]: This shouldn't be needed. the library is added on the node page already

**Comment by pookmish** [comment]
Assigned to joe to split out the node vs pattern styles

**Comment by josephgknox** [comment]
Thanks for the feedback, @pookmish!

File: themes/math_research_center/patterns/templates/node-simple/scss/node-simple.scss [hunk 2]
```diff
@@ -1,2 +1,32 @@
...
+// Decanter
+@import
+  'decanter-no-markup';
+
+// Stanford Math Research Center
+@import
+  '../utilities/mixins/mixins';
+
+.simple-node {
+  .field-s-visitor-photo {
```
Comment (pookmish) [inline themes/math_research_center/patterns/templates/node-simple/scss/node-simple.scss]: This is a node specific field. only the classes in the node-simple.html.twig should be used here.

**Review by pookmish — CHANGES_REQUESTED** [review]

**Review by pookmish — APPROVED** [review]

**Pull request merged**